src/__tests__/lib/rules/prefer-to-have-value.js

```diff
@@ -15,14 +15,34 @@ import * as rule from "../../../rules/prefer-to-have-value";
 // Tests
 //------------------------------------------------------------------------------
 
-const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2015 } });
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2017 } });
 
 const errors = [{ messageId: "use-to-have-value" }];
-ruleTester.run("prefer-empty", rule, {
+ruleTester.run("prefer-to-have-value", rule, {
   valid: [
     `expect(element).toHaveValue('foo')`,
     `expect(element.value).toBeGreaterThan(2);`,
     `expect(element.value).toBeLessThan(2);`,
+
+    `const element = document.getElementById('asdfasf');
```

Comment:
for now this is fine, but it would be nice to at some point support document queries as well as RTL. :)

Reply:
Yeah, my initial idea was to create a list of all the DOM api query functions but then it occurred to me that you can also get a reference to a dom element via a lot of other keys like `first/lastElementChild`, `children` etc. so I figured probably best to leave that to be implemented as its own piece